### PR TITLE
Set option name to json output

### PR DESCRIPTION
**Pull request by marsam**
This avoids shadowing the value of the option json with the json module

### CODE REVIEW

File: proselint/command_line.py [hunk 1]
```diff
@@ -113,9 +113,9 @@ def clear_cache():
         shell=True)
 
 
-def show_errors(filename, errors, json=False, compact=False):
+def show_errors(filename, errors, json_output=False, compact=False):
```
Comment (suchow) [inline proselint/command_line.py]: Can you rename this to `output_json`?

File: proselint/command_line.py [hunk 2]
```diff
@@ -158,13 +158,13 @@ def show_errors(filename, errors, json=False, compact=False):
 @click.option('--debug', '-d', is_flag=True)
 @click.option('--clean', '-c', is_flag=True)
 @click.option('--score', '-s', is_flag=True)
-@click.option('--json', '-j', is_flag=True)
+@click.option('--json', '-j', 'json_output', is_flag=True)
```
Comment (suchow) [inline proselint/command_line.py]: What does this change do?

**Comment by suchow** [comment]
Thanks! I left two small line comments with a suggested change and a question.

File: proselint/command_line.py (hunk 2 above)
Comment (marsam) [inline proselint/command_line.py]: Changes the [parameter name](http://click.pocoo.org/5/parameters/#parameter-names) from the default `json` to `json_output`

**Comment by marsam** [comment]
I've changed the parameter it to `output_json`

**Comment by suchow** [comment]
Great, thank you.

**Pull request merged**